content/webapp/components/Card/Card.tsx

```diff
@@ -20,6 +20,7 @@ type Props = {
 export const CardOuter = styled.a.attrs<{ 'data-gtm-trigger'?: 'card_link' }>({
   'data-gtm-trigger': 'card_link',
 })`
+  container: card / inline-size;
```

Comment:
we could call the container something slightly more generic than `card` in the event that we want to reuse this logic in other non-card components

Reply:
I feel like `card` makes sense for now, since we use it in line with `cardSizes`, right? and those are unlikely to be reused?